src/main/java/core/basesyntax/ReadWriteList.java

```diff
@@ -10,16 +11,36 @@
     private ReadWriteLock lock = new ReentrantReadWriteLock();
 
     public void add(E element) {
-        // write your code here
+        Lock writeLock = lock.writeLock();
+        writeLock.lock();
+        try {
+            list.add(element);
+        } finally {
+            writeLock.unlock();
+        }
     }
 
     public E get(int index) {
-        // write your code here
-        return null;
+        Lock writeLock = lock.writeLock();
```

Comment:
Read operations should use readLock() to be able to perform read jobs simultaneously.
With current implementation every thread waits until other finishes it's job so overall performance in similar to single-threaded